coax/proba_dists/_normal.py

```diff
@@ -48,15 +48,34 @@ class NormalDist(BaseProbaDist):
         Setting a finite range for :code:`clip_reals` ensures that the sigmoid doesn't fully
         saturate.
 
+    clip_logvar : pair of floats, optional
+
+        The range of values to allow for the log-variance of the distribution.
+
+    squash : boolean, optional
```

Comment:
If you do prefer to have this in the definition of the probability distribution, please could you create a new class like `SquashedNormalDist`. You could probably just wrap `NormalDist` and apply the changes like:

```
class SquashedNormalDist(BaseProbaDist):

  def __init__(...):

    normal_dist = NormalDist(...)

    def sample(dist_params, rng):
      X = normal_dist.sample(dist_params, rng)
      return jnp.tanh(X) * self._scale + self._offset

    ...

```

Reply:
This wrapper does what you're looking for: [`BoxActionsToReals`](https://coax.readthedocs.io/en/latest/coax/wrappers.html#coax.wrappers.BoxActionsToReals), i.e. it provides an interface such that the (original) env receives squashed actions, while the policy outputs non-squashed normal variates.

Reply:
I wanted to use a probability distribution because the log-likelihood `log_proba` has to be adjusted to account for the missing probability mass due to the squashing.

Reply:
Also, the clipping and squashing are mutual exclusive as of now, So I wanted to keep both in the same place. I will introduce the `SquashedNormalDist` class which does not perform clipping and warns if the clipping kwargs are set.

Reply:
Thanks so much!